src/mono/nuget/Microsoft.NET.Runtime.MonoTargets.Sdk/Microsoft.NET.Runtime.MonoTargets.Sdk.pkgproj

```diff
@@ -5,6 +5,10 @@
     <PackageDescription>Provides the tasks+targets, for consumption by mono-based workloads</PackageDescription>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
```

Comment:
I can't find the suggestion that @akoeplinger had, but the gist of his feedback was that this will work for net6.0, but should have something more generic.

Reply:
Is there a more generic TFM we can use? Or should we be checking something other than TargetFramework?

Reply:
I knew something was off...  The pkgproj is not the place that needs to pull in the `PackageReference`.  The RuntimeConfig task project needs something like:

```
<ItemGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkForNETCoreTasks)'">
    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
  </ItemGroup>
<ItemGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkForNETFrameworkTasks)'">
    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" PrivateAssets="All" />
  </ItemGroup>
```

And the `GetFilesToPackage` needs:

```
<FilesToPackage Include="$(OutputPath)%(_PublishFramework.Identity)\*.dll" TargetPath="tasks\%(_PublishFramework.Identity)" />
```